packages/vue-components/src/SiteNavButton.vue

```diff
@@ -5,7 +5,13 @@
     :portal-name="portalName"
   >
     <template #navMenuIcon>
-      <span :class="['glyphicon', 'toggle-site-nav-button']"></span>
+      <div
+        :class="['toggle-page-site-button']"
```

Comment:
Would `toggle-site-nav-button` be better? Similar to the one given to page nav.

Reply:
I changed back to the original `toggle-site-nav-button`. Forgot to change back while editing the code😅